ReactWindows/ReactNative.Shared/Reflection/EnumHelpers.cs

```diff
@@ -41,6 +39,30 @@ public static T Parse<T>(string value)
             return Parse<T>(value);
         }
 
+        private static Dictionary<string, object> EnumToDictionary(Type type)
+        {
+            var result = new Dictionary<string, object>();
+
+            var names = Enum.GetNames(type);
+            var values = Enum.GetValues(type);
+
+            for (int i = 0; i < values.Length; ++i)
+            {
+                string name = names[i];
+                object value = values.GetValue(i);
+
+                result.Add(Normalize(name), value);
+
+                var enumMemberAttribute = type.GetField(name).GetCustomAttributes(typeof(EnumMemberAttribute), true).SingleOrDefault();
+                if (enumMemberAttribute != null)
```

Comment:
That's true that Reflection brings in an overhead, but in this case, due to the fact that this method is called only once per enum, I think it's too small to have a significant impact on performance.